MAINTAINERS

```diff
@@ -17,3 +17,7 @@ rbrtbnfgl, Roberto Bonafiglia, roberto.bonafiglia@suse.com
 ShylajaDevadiga, Shylaja Devadiga, shylaja.devadiga@suse.com
 thomasferrandiz, Thomas Ferrandiz, thomas.ferrandiz@suse.com
 VestigeJ, Justin Janes, justin.janes@suse.com
+
+# Community Management
+OrlinVasilev, Orlin Vasilev, orlin@orlix.org
```

Comment:
Please use your `@suse.com` email address if opening this PR in your capacity as a SUSE employee.

Reply:
fixed, thanks :)